prowler/providers/gcp/gcp_provider.py

```diff
@@ -73,8 +81,10 @@ def __init__(
         accessible_projects = self.get_projects()
         if not accessible_projects:
             logger.critical("No Project IDs can be accessed via Google Credentials.")
-            # TODO: add custom exception once we have the GCP exceptions
-            raise SystemExit
+            raise GCPNoAccesibleProjectsError(
+                file=__file__,
+                original_exception="No Project IDs can be accessed via Google Credentials.",
```

Comment:
```suggestion
                message="No Project IDs can be accessed via Google Credentials.",
```

Why not in this way?

Reply:
This comment applies to the rest of the file with the same logic while creating exceptions.

Reply:
Because the point of this type of exceptions is to have this exceptions is to have "static" values for all the exceptions fields except the file and the original exception. I could get your point but if at the end the original_exception will appear too on the log, why to change the message?